evalml/tests/component_tests/test_components.py

```diff
@@ -813,7 +816,7 @@ def test_transformer_transform_output_type(X_y_binary):
     ]
 
     for component_class in _all_transformers():
-        if component_class == PolynomialDetrender:
+        if component_class in [PolynomialDetrender, LogTransformer]:
             # Skipping because this test is handled in test_polynomial_detrender
```

Comment:
can we update this comment to say these tests are handled in their respective test files?